lib/apiservers/portlayer/swagger.json

```diff
@@ -2654,8 +2654,8 @@
 			],
 			"properties": {
 				"Stores": {
-					"type": "object",
-					"additionalProperties": {
+					"type": "array",
```

Comment:
Some changes on the request side will also be needed for this change. I took a look and the code should still work fine, but the `VolumeStoreMap` var needs to be renamed. right here is where it might need a quick look: https://github.com/vmware/vic/blob/54921dd5b4ca95587db54792cdac8c046287bce7/lib/apiservers/engine/backends/system.go#L327-L341